ui/component/or-asset-viewer/src/index.ts

```diff
@@ -1038,6 +1040,7 @@ export class OrAssetViewer extends subscribe(manager)(translate(i18next)(LitElem
         this.addEventListener(OrChartEvent.NAME, () => OrAssetViewer.generateGrid(this.shadowRoot));
         this.addEventListener(OrAttributeHistoryEvent.NAME, () => OrAssetViewer.generateGrid(this.shadowRoot));
         this.addEventListener(OrEditAssetModifiedEvent.NAME, (ev: OrEditAssetModifiedEvent) => this._onAssetModified(ev.detail));
+        this.addEventListener('keyup', () => this._assetModified = true)
```

Comment:
It's really just about this. It does listen to any keyup happening in the assetviewer. I tried with change but there's no getting out of blurring (defeating the purpose of this ticket) and a cleaner keypress event is unusable because it doesn't include backspace.
Could scope this a bit better by putting custom events on all the or-attribute-input instances in the panel, but that wouldn't include other fields on the page. Thought this would be better in general.